tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/AcceptedGiftTypes.kt

```diff
@@ -0,0 +1,24 @@
+package dev.inmo.tgbotapi.types
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * This object describes the types of gifts that can be gifted to a user or a chat.
+ *
+ * @param unlimitedGifts True, if unlimited regular gifts are accepted
+ * @param limitedGifts True, if limited regular gifts are accepted
+ * @param uniqueGifts True, if unique gifts or gifts that can be upgraded to unique for free are accepted
+ * @param premiumSubscription True, if a Telegram Premium subscription is accepted
+ */
+@Serializable
+data class AcceptedGiftTypes(
```

Comment:
Looks like it must be `sealed interface` with four objects-realizations. In case if only one value can be true at the same time - I am right.

Besides, values in constructor do not have default values. Is it suitable to official docs? I think that here everywhere were `false` by default

Reply:
Official docs don't specify defaults, so it is assumed that every value is provided. I'll add defaults just in case.